test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -157,23 +170,13 @@ var _ = Describe("[rfe_id:28761][performance] Updating parameters in performance
 				Enabled: pointer.Bool(true),
 			}
 
-			By("Verifying that mcp is ready for update")
-			mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
-
-			spec, err := json.Marshal(profile.Spec)
-			Expect(err).ToNot(HaveOccurred())
-
-			By("Applying changes in performance profile and waiting until mcp will start updating")
-			Expect(testclient.Client.Patch(context.TODO(), profile,
-				client.RawPatch(
-					types.JSONPatchType,
-					[]byte(fmt.Sprintf(`[{ "op": "replace", "path": "/spec", "value": %s }]`, spec)),
-				),
-			)).ToNot(HaveOccurred())
-			mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdating, corev1.ConditionTrue)
+			By("Updating the performance profile")
+			profiles.UpdateWithRetry(profile)
+			testlog.Infof("Applying changes in performance profile and waiting until %s will start updating", resourcePool)
+			profilesupdate.WaitForTuningUpdating(context.TODO(), profile)
 
-			By("Waiting when mcp finishes updates")
-			mcps.WaitForCondition(performanceMCP, machineconfigv1.MachineConfigPoolUpdated, corev1.ConditionTrue)
+			testlog.Infof("Waiting when %s finishes updates", resourcePool)
```

Comment:
Why did we replace `By` with `testlog.Infof`?

Reply:
@SargunNarula can you replace this with By(fmt.Sprintf instead of testlog.Info

Reply:
The motivation was to keep log statements consistent. Revised with By().